Makefile

```diff
@@ -0,0 +1,167 @@
+.DEFAULT_GOAL		:=help
+SHELL				:=/bin/bash
+
+.PHONY: help
+help: ## Show this help
+	@grep -E '^[a-zA-Z\.\-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+##@ Set environment and corresponding configuration
+.PHONY: dev
+dev:
+	$(eval DEPLOY_ENV=dev)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
+	$(eval RESOURCE_NAME_PREFIX=s165d01)
+	$(eval ENV_SHORT=dv)
+	$(eval ENV_TAG=dev)
+	$(eval NAME_ENV=${DEPLOY_ENV})
+	$(eval RESOURCE_ENV=${ENV_SHORT})
+
+.PHONY: test
+test:
+	$(eval DEPLOY_ENV=test)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-test)
+	$(eval RESOURCE_NAME_PREFIX=s165t01)
+	$(eval ENV_SHORT=ts)
+	$(eval ENV_TAG=test)
+	$(eval NAME_ENV=${DEPLOY_ENV})
+	$(eval RESOURCE_ENV=${ENV_SHORT})
+
+.PHONY: preprod
+preprod:
+	$(eval DEPLOY_ENV=preprod)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-test)
+	$(eval RESOURCE_NAME_PREFIX=s165t01)
+	$(eval ENV_SHORT=pp)
+	$(eval ENV_TAG=pre-prod)
+	$(eval NAME_ENV=${DEPLOY_ENV})
+	$(eval RESOURCE_ENV=${ENV_SHORT})
+
+.PHONY: production
+production:
+	$(eval DEPLOY_ENV=production)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-production)
+	$(eval RESOURCE_NAME_PREFIX=s165p01)
+	$(eval ENV_SHORT=pd)
+	$(eval ENV_TAG=prod)
+	$(eval AZURE_BACKUP_STORAGE_ACCOUNT_NAME=s165p01aytqdbbackuppd)
+	$(eval AZURE_BACKUP_STORAGE_CONTAINER_NAME=aytq)
+	$(eval NAME_ENV=${DEPLOY_ENV})
+	$(eval RESOURCE_ENV=${ENV_SHORT})
+	$(eval CONSOLE_OPTIONS=--sandbox)
+
+.PHONY: review-init
+review-init:
+	$(if $(pr_id), , $(error Missing environment variable "pr_id"))
+	$(eval ENV_TAG=dev)
+
+.PHONY: review
+review: review-init set-azure-resource-group-tags
+	$(eval DEPLOY_ENV=review)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
+	$(eval RESOURCE_NAME_PREFIX=s165d01)
+	$(eval ENV_SHORT=rv)
+	$(eval env=-pr-$(pr_id))
+	$(eval backend_config=-backend-config="key=review/review$(env).tfstate")
```

Comment:
Should we be standardising the case of these to match all the others?

Reply:
Good spot, done